### ISSUE TITLE
Add List of Nearby Places

### ISSUE THREAD

**Issue by aej11a** [Peer-Stevens/peer#16]
Should include:

- some placeholder UI for the accessibility rating, which we will connect to actual data later

- native screen-reader should be able to read it
